app/src/components/cards/itemCard.tsx

```diff
@@ -40,6 +41,12 @@ const ItemCard = ({ id, isLargerThanLargeTablet, item }) => {
         id: `image-${id}`,
         isLazy: true,
         aspectRatio: "twoByOne",
+        fallbackSrc: "/noImage.png",
+        onError: (_event) =>
+          console.log(
```

Comment:
Use `console.warn` so it's more prominent in the console. (also but nitpick: use string interpolation).